CMakeLists.txt

```diff
@@ -145,18 +145,18 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   set_target_properties(ngx_http_datadog_module PROPERTIES LINK_DEPENDS ${LINKER_SCRIPT})
   target_link_options(ngx_http_datadog_module PRIVATE "-Wl,--version-script=${LINKER_SCRIPT}")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-  set(EXPORTED_SYMBOLS ${CMAKE_SOURCE_DIR}/nginx_module.exported)
+  set(EXPORTED_SYMBOLS ${CMAKE_SOURCE_DIR}/nginx_module.ld)
```

Comment:
I reused the `.ld` but it's up to you @cataphract

Reply:
The reason there are separate ones is because iirc the mac os linker doesn't support the version script format in the .ld file

Reply:
I was trying to build things on mac from `master` and was getting this error

`gmake[3]: *** No rule to make target '/Users/pablo.martinez/dev/nginx-datadog/nginx_module.exported', needed by 'ngx_http_datadog_module.so'.  Stop.`

Applying this suggestion makes my build succeed.